packages/core/test/errors/test/exception-handler.spec.ts

```
@@ -1,16 +1,15 @@
 import { ExceptionHandler } from '../../../errors/exception-handler';
 import { RuntimeException } from '../../../errors/exceptions/runtime.exception';
-import { InvalidMiddlewareException } from '../../../errors/exceptions/invalid-middleware.exception';
 
 describe('ExceptionHandler', () => {
   let instance: ExceptionHandler;
   beforeEach(() => {
     instance = new ExceptionHandler();
   });
   describe('handle', () => {
-    let logger;
+    let logger: { error: Function };
```

Biome

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)